DeepSpeech.py

```diff
@@ -3,179 +3,168 @@
 from __future__ import absolute_import
 # -*- coding: utf-8 -*-
 
+import os
+
+if 'TF_CPP_MIN_LOG_LEVEL' not in os.environ:
+    os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
```

Comment:
Forgot about this one - wanted to connect it to the `--log_level` parameter. Done now.